buildspec.yml

```diff
@@ -0,0 +1,36 @@
+version: 0.2
+
+phases:
+  install:
+    runtime-versions:
+      python: 3.7
+    commands:
+  pre_build:
+    commands:
+      - echo "[+] Running tests..."
```

Comment:
do tests actually run?

Reply:
no tests yet

Reply:
consider removing this pre_build step until then